resources/views/inputs/includes/input-label.blade.php

```diff
@@ -12,7 +12,10 @@
 
 <label
     for="{{ $id ?? $name }}"
-    class="items-center input-label @error($name) input-label--error @enderror"
+    @class([
+        'items-center input-label',
+        'input-label--error' => $errors->has('name'),
```

Comment:
```suggestion
        'input-label--error' => $errors->has($name),
```

Reply:
When I changed to this the `Username` label had the error class again, as it should